### DISCUSSION TITLE
Diagonal Exclusion Zone

### DISCUSSION THREAD

**Discussion by mkammes** [djdd87/SynoAI#133] [Q&A] [answered]
Hello!

I have 2 cameras mounted outside of my house. In order to not record my neighbors (and folks walking down the street), I have the cameras tilted downward, which causes the street to run diagonally through the image. I'm trying to have most of the street excluded.

Here is an example of 1 camera's parameters for the exclusion zone:

```
"Exclusions": [
           {
            "Start": { "X": 290, "Y": 0 },
            "End": { "X": 3840, "Y": 1900 }
           }
         ]
```

Ends up with a box, rather than a diagonal line for the exclusion zone. What values/techniques can I use to create a diagonal exclusion line?

![Driveway Left_2022_06_16_14_12_46_155_1](https://user-images.githubusercontent.com/8660699/174169139-363a61b3-3eb5-499e-b274-5895e7e18d64.jpeg)

Thanks!

**Comment by djdd87** [comment, marked as answer]
Hi, the exclusion zones are only boxes. For a triangle, it'd need multiple points, which there's no support for at the moment.

You could create multiple boxes like this (without the gaps obviously, but I appreciate that's going to be a PITA):

![image](https://user-images.githubusercontent.com/22151993/174260446-f12c8bd0-739c-41f8-ae14-84621fa29f1f.png)

**Comment by mkammes** [comment]
Understood. Thanks for the clarification!
